internal/server/server.go

```diff
@@ -167,6 +167,8 @@ func (s *Server) ListenAndServe(addr string) error {
 	mux.HandleFunc("/v1/auth/accounts/", s.HandleAccounts)
 	mux.HandleFunc("/v2/auth/snapshot", s.HandleSnapshotV2)
 	mux.HandleFunc("/v2/auth/snapshot/", s.HandleSnapshotV2)
+	mux.HandleFunc("/v2/auth/validate", s.HandleValidateSnapshotV2)
+	mux.HandleFunc("/v2/auth/validate-snapshot/", s.HandleValidateSnapshotV2)
```

Comment:
I think this one should have been `/v2/auth/validate/`? By the way the repetition here is because of the way that the mux handler in Go treats trailing slash (as 404)...

Reply:
Good catch!